.github/CONTRIBUTING.md

```diff
@@ -6,6 +6,21 @@ Thanks for your interest in contributing to Google Listings and Ads!
 
 Like the WooCommerce project, we want to ensure a welcoming environment for everyone. With that in mind, all contributors are expected to follow our [Code of Conduct](./CODE_OF_CONDUCT.md).
 
+If you wish to contribute code, please read the information in the sections below. Then [fork](https://help.github.com/articles/fork-a-repo/) Google Listings & Ads, commit your changes, and [submit a pull request](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) 🎉
+
+Google Listings & Ads is licensed under the GPLv3+, and all contributions to the project will be released under the same license. You maintain copyright over any contribution you make, and by submitting a pull request, you are agreeing to release that contribution under the GPLv3+ license.
```

Comment:
Are we sure about GPLv3+, or should it be GPLv3? 🤔

Reply:
Just notice that this is similar to [WooCommerce Core](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md), so I guess we are good.